syntax = "proto3";

package templates_stream;

//import "google/protobuf/empty.proto"; 
//import "google/protobuf/any.proto";
 
service TemplatesStream{
 rpc Changes(TemplatesStreamRequest) returns (stream TemplatesStreamResponse) {}
}

message TemplatesStreamRequest{}

message TemplatesStreamResponse{
  string template_id = 1;
    enum State{
     STOPPED = 0;
     STARTED = 1;
     PAUSED = 2;
     DELETED = 3;
     FAILED = 4;
     UNKNOWN = 5;
     NEW = 6;
    }
 State state = 2;
 string detail = 3; 
 repeated string next_actions = 4; 
 string kind = 5;
}
